delete.ps1

```diff
@@ -0,0 +1,161 @@
+            Line             = $ErrorObject.InvocationInfo.Line
+            ErrorDetails     = $ErrorObject.Exception.Message
+            FriendlyMessage  = $ErrorObject.Exception.Message
+        }
+
+        try {
+            #  Collect ErrorDetails
+            if ($ErrorObject.Exception.GetType().FullName -eq 'Microsoft.PowerShell.Commands.HttpResponseException') {
+                $httpErrorObj.ErrorDetails = $ErrorObject.ErrorDetails.Message
+
+            }
+            elseif ($ErrorObject.Exception.GetType().FullName -eq 'System.Net.WebException') {
+                if ($null -ne $ErrorObject.Exception.Response) {
+                    if ([string]::IsNullOrEmpty($ErrorObject.ErrorDetails.Message)) {
+
+                        $streamReaderResponse = [System.IO.StreamReader]::new($ErrorObject.Exception.Response.GetResponseStream()).ReadToEnd()
+                        if ($null -ne $streamReaderResponse) {
+                            $httpErrorObj.ErrorDetails = $streamReaderResponse
+                        }
+                    }
+                    else {
+                        $httpErrorObj.ErrorDetails = $ErrorObject.ErrorDetails.Message
+                    }
+                }
+            }
+            $errorDetailsObject = ($httpErrorObj.ErrorDetails | ConvertFrom-Json)
+            $httpErrorObj.FriendlyMessage = "$($errorDetailsObject.error.message)"
+
+        }
+        catch {
+            $httpErrorObj.FriendlyMessage = $httpErrorObj.ErrorDetails
+        }
+        Write-Output $httpErrorObj
+    }
+}
+#endregion
+
+try {
+    # Verify if [aRef] has a value
+    if ([string]::IsNullOrEmpty($($actionContext.References.Account))) {
+        throw 'The account reference could not be found'
+    }
+
+    $headers = @{
+        'Content-Type'        = 'application/json; charset=utf-8'
+        Accept                = 'application/json; charset=utf-8'
+        'X-FACILITOR-API-KEY' = $actionContext.Configuration.APIKey
+    }
+
+    Write-Verbose "Verifying if a Facilitor account for [$($personContext.Person.DisplayName)] exists"
+    try {
+        $splatGetUser = @{
+            Uri     = "$($actionContext.Configuration.BaseUrl)/api2/persons/$($actionContext.References.Account)"
+            Method  = 'GET'
+            Headers = $headers
+        }
+        $correlatedAccount = (Invoke-RestMethod @splatGetUser).person
+    }
+    catch {
+        # A '404' is returned if the entity cannot be found
+        if ($_.Exception.Response.StatusCode -eq 404) {
+            $correlatedAccount = $null
+        }
+        else {
+            throw
+        }
+    }
+
+    if ($null -ne $correlatedAccount) {
+        $action = 'DisableAccount'
+    }
+    else {
+        $action = 'NotFound'
+    }
+
+    # Process    
+    switch ($action) {
+        'DisableAccount' {
+            Write-Verbose "Disabling Facilitor account with accountReference: [$($actionContext.References.Account)]"
```

Comment:
Write-Verbose "Disabling Facilitor account with accountReference: [$($actionContext.References.Account)]"
moet dan denk ik ook "Delete Facilitor..." worden

Reply:
Actie die HelloID uitvoert is disable